playbook.md

```diff
@@ -178,17 +178,42 @@ Each story will also contain a list of acceptance criteria. Acceptance criteria
 For more information about writing good stories, read User Stories Applied by Mike Cohn. There is a copy of this book in the dxw library.
 
 
-#### Lifecycle of a story
-There are several states that a story has to go through in order to be deployed to production. We use tracking tools and physical story boards to keep track of which stage a given story is in.
+## Lifecycle of a story
+There are several states that a story has to go through in order to be deployed to production. We use tracking tools and physical story boards to keep track of which state a given story is in.
 
-* **Started**: A developer has started working on the story, making changes to the product to ensure that each acceptance criterion is met
-* **Code review**: The first developer has finished their work, and is ready for a second developer to [review](#Code reviews) the technical aspects of the story in a [pull request](#Pull request)
-* **Acceptance review**: The two developers have finished the review and the story is now merged, and ready for the delivery manager to review against its acceptance criteria
-* **Client review**: The delivery manager has confirmed that the story is done, and it's ready for a client to review
-* **Ready to deploy**: The client has accepted the story and it's ready to deploy
-* **Deployed**: The story is deployed to production
+### States
 
-#### Branching
+After a user need has been identified, a story is written to capture it and how we propose to meet that need with acceptance critera. Once it's written and prioritised as part of a sprint planning it will be ready to start.
+
+#### Unstarted
+- hasn't been started or isn't being actively developed yet and can be changed
+
+#### In progress
+- a developer or a pair are currently working on meeting the acceptance criteria
+
+#### Awaiting code review
+- a pull request has been opened
+- another developer is required to review the codes quality and merge into develop
+- this developer also tests the functionality with the aim of catching the majority of any bugs as early as possible and feedback them back onto the pull request
```

Comment:
@tomdxw that is a fair concern but we will arrange the process so that that won't happen. 

Current plan: if the support dev can't solve the issue it will be escalated to another dev to deal with on the following Friday, and the SLA will be configured to understand that and not to breach prematurely.

(hopefully this will not happen very often)